### PR TITLE
Add Github Issue templates

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  end
+end
+```
+
+Changes proposed in this pull request:
+* 
+* 
+* 
+
+@projecthydra/sufia-code-reviewers
```
Comment (escowles) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is this the right group to ping for CC pull requests?

Comment (jcoyne) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can you think of a better one?

Comment (escowles) [inline .github/PULL_REQUEST_TEMPLATE.md]: No -- maybe we should just add me and @tpendragon to that group?

**Comment by escowles** [comment]
I went ahead and added myself and @tpendragon to the Sufia Code Reviewers team.  We can always decide to adjust the team membership later on if that makes sense.

**Pull request merged**